lib/OpenQA/WebAPI/Controller/API/V1/Iso.pm

```diff
@@ -293,7 +302,8 @@ sub _generate_jobs {
             push @parents, _parse_dep_variable($ret->[$i]->{START_AFTER_TEST}, $ret->[$i]);
             push @parents, _parse_dep_variable($ret->[$i]->{PARALLEL_WITH},    $ret->[$i]);
             for my $p (@parents) {
-                $wanted{$p} = 1;
+                my ($t, $m) = @$p;
+                $wanted{$t . ':' . $m} = 1;
```

Comment:
maybe `join(':', @$p)` as we do not need $t and $m

Reply:
And don't use one-letter variable names (also in the other occurrences). It just makes the code harder to read and understand if you're not already familiar with it (especially in the dynamically typed Perl).